apps/calendar/js/bridge.js

```diff
@@ -91,10 +94,70 @@ var persistEvent = co.wrap(function *(event, action, capability) {
   return provider[action + 'Event'](event.data);
 });
 
-exports.getSetting = co.wrap(function *(id) {
-  var settingsStore = core.storeFactory.get('Setting');
-  return yield settingsStore.getValue(id);
-});
+exports.getSetting = function(id) {
```

Comment:
I disagree, there are cases where we just care about a one of read and it is *harder* to remove the listener afterwards... think of `getSetting` as `once` on a regular event emitter. - in most cases we should use the `observeSetting` tho..